apps/chat-e2e/src/ui/webElements/editInputActions.ts

```diff
@@ -0,0 +1,17 @@
+import { EditSelectors } from '@/src/ui/selectors/editSelectors';
+import { BaseElement } from '@/src/ui/webElements/baseElement';
+import { Locator, Page } from '@playwright/test';
+
+export class EditInputActions extends BaseElement {
+  constructor(page: Page, parentLocator: Locator, selector: string) {
+    super(page, `${selector} >> ${EditSelectors.actionButton}`, parentLocator);
+  }
+
+  public async clickTickButton() {
+    await this.getNthElement(1).click();
```

Comment:
I suppose it's better to use `data-qa` instead here and below

Reply:
good point, thank you! Fixed